# Unique identifier of the object in the world
string uid
# History of states of the object
h2sl/ObjectStateMessage[] state_history
# List of object properties and their values
h2sl/PropertyMessage[] properties
